dbt-on-astro/README.md

```diff
@@ -1,23 +1,34 @@
-# cosmos-demo
+# Project Overview
 
-This repo contains a dbt project and a set of Airflow DAGs showing how to run dbt in Airflow using [Cosmos](https://github.com/astronomer/astronomer-cosmos).
+Apache Airflow is one of the most widely-used engines for orchestrating Extract, Transform, and Load (ETL) jobs, especially for transformations using [dbt](https://www.getdbt.com). dbt is a framework to create reliable transformations to produce high-quality data for businesses, usually in analytical databases like Snowflake and BigQuery.
 
-To run this, you'll need:
+This project showcases using dbt and Airflow together with [Cosmos](https://github.com/astronomer/astronomer-cosmos), allowing users to deploy dbt in production with Airflow best-practices.
 
-- [The Astro CLI installed](https://docs.astronomer.io/astro/cli/overview)
-- Docker
+# Learning Paths
 
-## Setup
+To learn more about data engineering with Apache Airflow, dbt, and Cosmos, make a few changes to this project! For example, try one of the following:
 
-1. Clone this repo
-2. Run `astro dev start` to start the Airflow instance
+1. Use Postgres, MySQL, Snowflake, or another production-ready database instead of DuckDB
```

Comment:
Might be good to say "instead of local DuckDB" and include Motherduck: https://motherduck.com/ who have the commercial cloud DuckDB offering, since I assume that move would be easiest for users :)

Reply:
applied!